utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
```

Comment:
`obs` gets created whether there's already an existing observer or not. on line 36 it sets the new observer only if one doesn't exist. this could have the scenario where an observer is created and set and then a new observer is created but doesn't get set. there are two observers running, but only one is remembered. It may be more prudent to wrap the observer creation in the `if (!observer)` so it only gets created if one is needed.